skymp5-front/src/constructorComponents/chat/index.js

```diff
@@ -10,50 +10,68 @@ import Settings from './settings';
 import SendButton from './sendButton';
 import ChatInput from './input';
 
-const FULL_NON_RP_REGEX = /(.*?):\s*\(\((.*?)\)\)/gi;
-const NONRP_REGEX = /\(\((.*?)\)\)/gi;
-const ACTION_REGEX = /\*(.*?)\*/gi;
-const IS_DICES_MESSAGE = /#\{.{6}\}(.)+ #\{ffffff\}- (.)+/gi;
-
 const MAX_LENGTH = 700; // Max message length
 const TIME_LIMIT = 5; // Seconds
+const SHOUT_LIMIT = 60; // Seconds
+
+const SHOUTREGEXP = /№(.*?)№/gi;
```

Comment:
Why do we need this regexp if we have stack parser?

Reply:
It’s for frontend input logic: message with shout can only be sent once in 2 minutes + we count symbols inside a shout; it should be <= 100